src/mcl_3dl.cpp

```diff
@@ -908,18 +904,17 @@ class MCL3dlNode
       Vec3 acc_measure = acc / acc.norm();
       try
       {
-        tf::Stamped<tf::Vector3> in, out;
-        in.frame_id_ = msg->header.frame_id;
-        in.stamp_ = msg->header.stamp;
-        in.setX(acc_measure.x_);
-        in.setY(acc_measure.y_);
-        in.setZ(acc_measure.z_);
-        tfl_.transformVector(frame_ids_["base_link"], in, out);
-        acc_measure = Vec3(out.x(), out.y(), out.z());
-
-        tf::StampedTransform trans;
+        geometry_msgs::Vector3 in, out;
+        in.x = acc_measure.x_;
+        in.y = acc_measure.y_;
+        in.z = acc_measure.z_;
+        geometry_msgs::TransformStamped trans =
+            tfbuf_.lookupTransform(frame_ids_["base_link"], msg->header.frame_id, ros::Time(0));
+        tf2::doTransform(in, out, trans);
+        acc_measure = Vec3(out.x, out.y, out.z);
+
         // Static transform
-        tfl_.lookupTransform(frame_ids_["base_link"], msg->header.frame_id, ros::Time(0), trans);
+        trans = tfbuf_.lookupTransform(frame_ids_["base_link"], msg->header.frame_id, ros::Time(0));
```

Comment:
(this and https://github.com/at-wat/mcl_3dl/pull/208/files#r238511052 should be always static transform (base_link->imu_link) in most case. timestamp is not mandatory to be treated here.)